fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/temp/explain/DotExporter.java

```diff
@@ -192,43 +193,71 @@ private void renderInvisibleEdges(@Nonnull final ExporterContext context, final
         // on the same level but left to right.
         final Set<E> childrenEdges = network.inEdges(n);
 
-        final boolean needsInvisibleEdges = childrenEdges.stream().anyMatch(edge -> !edge.getDependsOn().isEmpty());
-
-        if (needsInvisibleEdges) {
-            final Optional<List<E>> orderedChildrenEdgesOptional =
-                    TopologicalSort.anyTopologicalOrderPermutation(childrenEdges, edge -> (Set<E>)edge.getDependsOn());
-            Verify.verify(orderedChildrenEdgesOptional.isPresent());
-            final List<E> orderedChildrenEdges = orderedChildrenEdgesOptional.get();
-
-            final ArrayList<N> childrenOperatorList = new ArrayList<>();
-            for (final E currentEdge : orderedChildrenEdges) {
-                final N currentChildNode = network.incidentNodes(currentEdge).nodeU();
-                if (currentChildNode instanceof PlannerGraph.ExpressionRefHeadNode) {
-                    final Set<N> refMembers = network.predecessors(currentChildNode);
-                    for (final N refMember : Sets.filter(refMembers, refMember -> refMember instanceof PlannerGraph.ExpressionRefMemberNode)) {
-                        childrenOperatorList.addAll(network.predecessors(refMember));
+        if (childrenEdges.stream().allMatch(edge -> edge.getDependsOn().isEmpty())) {
```

Comment:
Note to reviewer: This is some layout improvement for the internal explain, i.e. the explain that is only used for debugging purposes. Disregard changes as they are complicated and only improve the rendering of the boxes.